src/decode.h

```diff
@@ -250,7 +276,20 @@ static inline uint32_t sign_extend_b(const uint32_t x)
     return (int32_t) ((int8_t) x);
 }
 
+/* clear all block in the block map */
+static inline void block_map_clear(struct block_map *map)
```

Comment:
We may provide similar shortcut in `src/map.h` in another pull request.

Reply:
> We may provide similar shortcut in `src/map.h` in another pull request.

I'm not understand about this. Can you explain more details ?

Reply:
I meant that we can provide a macro or inline function to release and free a map object.